tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -630,7 +630,7 @@ public static function logScenario(BeforeScenarioScope $scope): void {
 	 * @throws Exception
 	 */
 	public static function logStep(BeforeStepScope $scope): void {
-		$step = $scope->getStep()->getType() . " " . $scope->getStep()->getText();
+		$step = $scope->getStep()->getKeywordType() . " " . $scope->getStep()->getText();
```

Comment:
We want to have `Given`, `When`, `Then` and also `And`
```suggestion
		$step = $scope->getStep()->getKeyword() . " " . $scope->getStep()->getText();
```